clojure/src/info/lotharschulz/armstrong_numbers/armstrong_numbers.clj

```diff
@@ -0,0 +1,49 @@
+(ns info.lotharschulz.armstrong_numbers.armstrong_numbers
+  (:import [java.Math]))
+
+(defn getNumberOfDigits [^Integer n]
+  {:pre [(not (nil? n)) (integer? n)]}
+  "Returns the :getNumberOfDigits of n if n is an integer and greater than -1"
+  (if (zero? n)
+    1
+    (if (pos? n )
+      (+ (int (Math/log10 n )) 1)
+      nil
+      )
```

Comment:
please refer to style guide for parenthesis formatting:
https://github.com/bbatsov/clojure-style-guide#gather-trailing-parens